howto/website-guidelines-checklist.md

```diff
@@ -5,8 +5,6 @@ As per the [CNCF Website Guidelines](https://github.com/cncf/foundation/blob/mas
 
 - [ ] 1. Website should be [hosted in an open source repo](./repo-setup.md)
   - [ ] Hosted in the same organization as the main project
-  - [ ] Setup [DCO](https://github.com/apps/dco) or CLA (DCO recommended)  
-   CNCF's IP policy (https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy) requires all projects to use either CLA (Contributor License Agreements) or [DCO (Developer Certificate of Origin)](https://github.com/apps/dco). Unless there's a strong necessity to use CLA, we encourage projects to use DCO as it's easier to setup and use.
```

Comment:
We may want to remove these points, but I'm not sure this is the PR to do that in.

Reply:
I think I resolved this by dropping the commit, but I'm gonna take a backup of the file just in case.